source/clear-linux/reference/bundles/openssh-server.rst

```diff
@@ -8,6 +8,46 @@ Remote users require a SSH service to be able to use an encrypted login
 shell. The first time OpenSSH starts, it generates the server SSH keys needed
 for the service.
 
+|CL| enables the `sshd.socket` unit, which will listen on port 22 by default and
+start the openssh service as required.
+
+Change Default Port
+===================
+In order to change the default listen port for the OpenSSH\* service, perform
+the following steps:
+
+#. Edit the sshd.socket unit file, provide the `ListenStream` option in the
+   `[Socket]` section with no value in order to remove the |CL| default port
+   value, then provide the `ListenStream` option again with the new default
+   port to listen. In this example, we change `ListenStream` to
+   listen on port 4200 instead of the |CL| default:
+
+   .. code-block:: console
```

Comment:
Code blocks with commands should use code-block:: bash. Console output should use code-block:: console.
Don't include the prompt.